src/ParseQuery.js

```diff
@@ -713,7 +722,8 @@ class ParseQuery {
     const controller = CoreManager.getQueryController();
     const params = {
       distinct: key,
-      where: this._where
+      where: this._where,
+      hint: this._hint ? this._hint : undefined,
```

Comment:
just `hint: this.hint` is fine

Reply:
The reason I did this was because I caused tests to fail because in the constructor this._hint is set to null (like readPreference)... so I can either take setting this._hint as null out of the constructor or keep it like this?

Reply:
You will know once you start writing tests.

Reply:
done!